src/react-apps/applications/runtime/src/features/form/containers/FormFiller.tsx

```diff
@@ -29,12 +32,20 @@ const FormFiller = (props: IFormFillerProps) => {
       header={
         props.applicationMetadata &&
           props.applicationMetadata.title[userLanguage] ? props.applicationMetadata.title[userLanguage] :
-        getLanguageFromKey('general.ServiceName', props.textResources)
+          getLanguageFromKey('general.ServiceName', props.textResources)
       }
       step={processStep}
     >
       <div>
-        <Render />
+        {props.isLoading === false ? (
```

Comment:
```suggestion
        {!props.isLoading ? (
```

Reply:
The initialState is "null". Since we don't have the "queue" yet which dispatches in an orderly fashion and includes startLoading action, we need to check for falsy right now.

Reply:
👍